.travis.yml

```diff
@@ -17,7 +17,10 @@ install:
 
 script:
   - poetry run pytest -vv
-
+  
+before_deploy:
+- poetry config pypi-token.pypi $PYPI_API_TOKEN
```

Comment:
do you need 
  on:
    tags: true

for this too?

Reply:
I don't think so I was following the travis yaml from dictionaryutils https://github.com/uc-cdis/dictionaryutils/blob/master/.travis.yml

Reply:
yeah I found the docs to verify ` These will only be triggered if Travis CI is actually pushing a release.`

https://docs.travis-ci.com/user/deployment/pypi/#running-commands-before-and-after-release